Makefile

```diff
@@ -4,8 +4,8 @@ CURRENT_DIR     = $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 TF_EXAMPLES     = $(sort $(dir $(wildcard $(CURRENT_DIR)examples/*/)))
 TF_MODULES  = $(sort $(dir $(wildcard $(CURRENT_DIR)modules/*/)))
 
-TF_VERSION      = 0.11.14
-TF_DOCS_VERSION = 0.6.0-0.2
+TF_VERSION      = 0.12.2
```

Comment:
What about using `light` here which will automatically hold the latest version (which will increment the patch levels automatically)?